worlds/alttp/Regions.py

```diff
@@ -468,19 +468,19 @@ def mark_light_world_regions(world, player: int):
      'Hobo': (0x33e7d, 0x186368, False, 'with the hobo'),
      'Lost Woods Hideout': (0x180000, 0x186348, False, 'near a thief'),
      'Lumberjack Tree': (0x180001, 0x186349, False, 'in a hole'),
-     'Cave 45': (0x180003, 0x18634b, False, 'alone in a cave'),
-     'Graveyard Cave': (0x180004, 0x18634c, False, 'alone in a cave'),
-     'Checkerboard Cave': (0x180005, 0x18634d, False, 'alone in a cave'),
+     'Cave 45': (0x180003, 0x18634b, False, 'in a T-shaped cave'),
+     'Graveyard Cave': (0x180004, 0x18634c, False, 'in a narrow cave'),
+     'Checkerboard Cave': (0x180005, 0x18634d, False, 'guarded by many blocks'),
      'Mini Moldorm Cave - Far Left': (0xeb42, 0x186316, False, 'near Moldorms'),
      'Mini Moldorm Cave - Left': (0xeb45, 0x186319, False, 'near Moldorms'),
      'Mini Moldorm Cave - Right': (0xeb48, 0x18631c, False, 'near Moldorms'),
      'Mini Moldorm Cave - Far Right': (0xeb4b, 0x18631f, False, 'near Moldorms'),
      'Mini Moldorm Cave - Generous Guy': (0x180010, 0x18635a, False, 'near Moldorms'),
      'Ice Rod Cave': (0xeb4e, 0x186322, False, 'in a frozen cave'),
-     'Bonk Rock Cave': (0xeb3f, 0x186313, False, 'alone in a cave'),
+     'Bonk Rock Cave': (0xeb3f, 0x186313, False, 'in a cave with an altar'),
      'Library': (0x180012, 0x18635c, False, 'near books'),
      'Potion Shop': (0x180014, 0x18635d, False, 'near potions'),
-     'Lake Hylia Island': (0x180144, 0x186353, False, 'on an island'),
+     'Lake Hylia Island': (0x180144, 0x186353, False, 'on an island in the water'),
```

Comment:
floating in the water?

Reply:
the item isn't floating in the actual water though. could do island in the lake though possibly?